setups/fullnode/systemd/setup.sh

```diff
@@ -14,10 +14,13 @@ G='\033[0;32m' # Green
 NC='\033[0m'   # No color
 
 # Check dependencies
-if [ "$(uname -s)" != "Linux" ]; then
-	red "[ERROR] This script is for systemd so will only work on Linux."
+case "$(uname -s)" in
+Linux | Darwin) ;;
```

Comment:
In the rest of the script, the OS are put in "". Will this work correctly here, and why the difference?